src/styles/utils.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { ReactNode } from "react";
+
+interface VerticalFlexboxProps {
+    children: ReactNode;
+    className?: string;
+    gap?: number;
+    alignItems?: string;
+    justifyContent?: string;
+}
+
+const VerticalFlexbox: React.FC<VerticalFlexboxProps> = ({
```

Comment:
As a component, should this live inside a `styles` directory?  I suggest putting it someplace where utils or other components live.

Reply:
Fair point, I'll make a folder for utility components, that's more accurate